### ISSUE TITLE
Multiformats organization captain

### ISSUE THREAD

**Issue by lgierth** [multiformats/multiformats#29]
The question came up who's captaining the multiformats organization (i.e. not the individual projects, but the organization). @RichardLitt said he's captaining.

Where did this question come from, is there any reponsibility that isn't covered yet? cc @VictorBjelkholm

**Comment by RichardLitt** [comment]
To be fair, I am captaining the multiformats/multiformats repo - which is the main repo for dealing with the other repos. This is like saying I am captaining ipfs/ipfs, which doesn't mean I captain the IPFS project. 

There is a list of captains for each repo, here: http://github.com/multiformats/multiformats.

**Comment by VictorBjelkholm** [comment]
Hey, I raised this originally. I understand Richard is the captain over multiformats/multiformats, but I would like to have someone making a implementation or the specification of a multiformat to help out with designing with what goes on the website and so on (with libp2p I spoke with David, with ipld I spoke with Nicola). I'm not sure it makes sense if me and Richard sits down and talks about what needs to be on the multiformats website.

**Comment by RichardLitt** [comment]
I wish I had taken better care to make sure that the notes about the multiformats website - specifically - were captured in multiformats/website. We have already covered what should be on the website a couple of times, and I think that a lot of the work you've been doing is coalescing that information into actionable goals (and thank you for that!). I've just tried to find some of those notes, and wasn't able to come up with anything substantive. 

I think that a conversation with @jbenet, you, and me may be the best thing to do, going forward with this. Perhaps @flyingzumwalt, too.
